great_expectations/expectations/expectation.py

```diff
@@ -338,7 +348,7 @@ class Expectation(metaclass=MetaExpectation):
         "result_format",
     )
     default_kwarg_values: ClassVar[
-        dict[str, bool | str | float | RuleBasedProfilerConfig | None]
+        Dict[str, Union[bool, str, float, RuleBasedProfilerConfig, None]]
```

Comment:
I think pydantic ignores `ClassVar` items so you should still be able to use future annotations here.